app/views/layouts/_generic.njk

```diff
@@ -51,5 +51,18 @@
   {% endif %}
 
   <script src="/public/all.js"></script>
-  <script>window.GOVUKFrontend.initAll()</script>
+  <script>window.GOVUKFrontend.initAll({
```

Comment:
So I know I'm understanding correctly, can you explain/describe how a service that has to provide multiple different translations of it's site use this in practice? Would they need to swap out this translation object depending on the locale of the site?

Reply:
Nice. Hopefully if there's any issues with that approach they'll be flagged when we get service teams to test this. It might be a good idea to get GOV.UK to test this (as a good example of a team needing to translate into many many languages)